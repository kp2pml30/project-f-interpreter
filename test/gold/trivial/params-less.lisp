((lambda (a) (return 10)))
