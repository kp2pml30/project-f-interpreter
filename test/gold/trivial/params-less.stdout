Error: wrong arguments number got 0, expected 1 for <func {"overallPos":1,"line":1,"offset":1}>; stack:
	{"overallPos":0,"line":1,"offset":0}